common/diagnostics.py

```diff
@@ -195,6 +186,38 @@ def collect_diagnostics():
     return result
 
 
+def _get_qt_information():
+    """Collect Version and Theme information from Qt.
+
+    If environment variable DISPLAY is set a temporary QApplication instances
+    is created.
+    """
+    try:
+        import PyQt5.QtCore
+        import PyQt5.QtGui
+        import PyQt5.QtWidgets
+    except ImportError:
+        return '(Can not import PyQt5)'
+
+    # Themes
+    theme_info = {}
+    if 'DISPLAY' in os.environ:
+        qapp = PyQt5.QtWidgets.QApplication([])
```

Comment:
I am not sure if using the DISPLAY env var is working in case of Wayland. How about simply using a try-catch clause (which is missing here anyhow I think since establishing a `QApplication` instance may fail somehow (we had some related issues with that)...

Reply:
`try...except` was my first try. But Qt doesn't raise anything here but just crash the whole Python instance when `QApplication()` without running X. Any idea how I can take Wayland into account here?

Reply:
Or much more "dirty" just using an external python call and checking its result.

    python3 -c "from PyQt5.QtWidgets import QApplication; QApplication([])"